cmd/prune-junit-xml/prunexml.go

```diff
@@ -29,33 +29,30 @@ import (
 func main() {
 	maxTextSize := flag.Int("max-text-size", 1, "maximum size of attribute or text (in MB)")
 	flag.Parse()
+	for _, path := range flag.Args() {
```

Comment:
git diff is struggling. this is just de-indent and drop the `if flag.NArg() > 0`, `for _, path := range flag.Args()` is already a no-op if len(Args()) is 0.

Reply:
yamlfmt has the same unnecessary check,  https://github.com/kubernetes/kubernetes/pull/118066